lib/Models/ImageryLayerCatalogItem.js

```diff
@@ -44,7 +44,7 @@ var ImageryLayerCatalogItem = function(terria) {
     /**
      * Gets or sets the opacity (alpha) of the data item, where 0.0 is fully transparent and 1.0 is
      * fully opaque.  This property is observable.
-     * @type {Number}
+     * @type {Number|String}
```

Comment:
You wouldn't think so, but: https://github.com/NICTA/nationalmap/blob/master/datasources/09_Tasmanian_Government.json#L8